docs/core/install/macos.md

```diff
@@ -1,95 +1,93 @@
 
 > [!div class="button"]
-> [Download .NET Core Runtime](https://dotnet.microsoft.com/download/dotnet-core)
+> [Download .NET Runtime](https://dotnet.microsoft.com/download/dotnet-core)
 
 ## SDK information
 
-The SDK is used to build and publish .NET Core apps and libraries. Installing the SDK includes both [runtimes](#runtime-information): ASP.NET Core and .NET Core.
-
-> [!div class="button"]
-> [Download .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core)
+The SDK is used to build and publish .NET apps and libraries. Installing the SDK includes both [runtimes](#runtime-information): ASP.NET Core and .NET.
 
 ## Dependencies
 
-.NET Core is supported on the following macOS releases:
+.NET is supported on the following macOS releases:
 
 > [!NOTE]
 > A `+` symbol represents the minimum version.
 
-| .NET Core Version | macOS                 | Architectures |     |
+| .NET Core Version | macOS                 | Architectures | More information    |
 | ----------------- | --------------------- | --------------| --- |
+| 5.0               | High Sierra (10.13+)  | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-supported-os.md) |
 | 3.1               | High Sierra (10.13+)  | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-supported-os.md) |
 | 3.0               | High Sierra (10.13+)  | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md) |
 | 2.2               | Sierra (10.12+)       | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md) |
 | 2.1               | Sierra (10.12+)       | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md) |
 
-Beginning with macOS Catalina (version 10.15), all software built after June 1, 2019 that is distributed with Developer ID, must be notarized. This requirement applies to the .NET Core runtime, .NET Core SDK, and software created with .NET Core.
+Beginning with macOS Catalina (version 10.15), all software built after June 1, 2019 that is distributed with Developer ID, must be notarized. This requirement applies to the .NET runtime, .NET SDK, and software created with .NET.
 
-The installers for .NET Core (both runtime and SDK) versions 3.1, 3.0, and 2.1, have been notarized since February 18, 2020. Prior released versions aren't notarized. If you run a non-notarized app, you'll see an error similar to the following image:
+The installers for .NET (both runtime and SDK) 5.0 and .NET Core 3.1, 3.0, and 2.1, have been notarized since February 18, 2020. Prior released versions aren't notarized. If you run a non-notarized app, you'll see an error similar to the following image:
```

Comment:
Parenthetical gets a little awkward about what it applies to, maybe pull earlier?

```suggestion
The runtime and SDK installers for .NET 5.0 and .NET Core 3.1, 3.0, and 2.1, have been notarized since February 18, 2020. Prior released versions aren't notarized. If you run a non-notarized app, you'll see an error similar to the following image:
```